server/src/utils/enums/ErrorMessages.ts

```diff
@@ -23,7 +22,23 @@ export enum ErrorMessages {
 	// Forgot Password
 	EMAIL_NOT_FOUND_ERROR_MESSAGE = 'This account does not exist. Please try again.',
 
-	// Validation Args
+	// Validation Member
 	USERNAME_MUST_BE_LONG_ERROR_MESSAGE = 'The username must be at least 3 character long.',
-	PASSWORD_FORMAT_ERROR_MESSAGE = 'The password must have at least 8 characters, one upper case, one lower case, one number and one special character.'
+	PASSWORDS_DO_NOT_MATCH_ERROR_MESSAGE = 'Passwords do not match',
+	PASSWORD_FORMAT_ERROR_MESSAGE = 'The password must have at least 8 characters, one upper case, one lower case, one number and one special character.',
+
+	// Validation Project
+	PROJECT_NOT_FOUND = 'Project not found',
+	MEMBER_NOT_FOUND = 'Member not found',
+
+	// Validation Language
+	LANGUAGE_NOT_FOUND = 'Language not found',
+	LANGUAGE_ALREADY_EXISTS = 'Language already exists',
+
+	// Validation FileProject
+	FILE_NOT_FOUND = 'File not found',
+
+	// Validation SandpackTemplate
+	SANDPACK_TEMPLATE_ALREADY_EXISTS = 'Sandpack template already exists',
```

Comment:
It's not necessary to tell user we use Sandpack, "code template" is okay i think